### ISSUE TITLE
Support for latest FP PRO ^3.11

### ISSUE THREAD

**Issue by Harm-Nullix** [fingerprintjs/fingerprintjs-pro-spa#85]
Latest fingerprint pro is ^3.11

lock file states pro-spa depends on ^3.9

Please update it to reflect the newest updates.
It would be nice if "latest" release would always be in sync with the "latest" fp-pro.
Or configurable.

yarn.lock
```
"@fingerprintjs/fingerprintjs-pro-spa@1.3.1":
  version "1.3.1"
  resolved "https://registry.yarnpkg.com/@fingerprintjs/fingerprintjs-pro-spa/-/fingerprintjs-pro-spa-1.3.1.tgz#4159fd57b03378703a08ff3bcc21b196939607af"
  integrity sha512-1k+85IYh1bBDJY/syHzBmTzKKaPGQkpq8GO9+vf7MHdqlmIiLAcutaHMUXey9DxylKm93qCHRY/1fCOOQ/LiTA==
  dependencies:
    "@fingerprintjs/fingerprintjs-pro" "^3.9.2"
    tslib "^2.6.2"

"@fingerprintjs/fingerprintjs-pro@^3.9.2":
  version "3.9.2"
  resolved "https://registry.yarnpkg.com/@fingerprintjs/fingerprintjs-pro/-/fingerprintjs-pro-3.9.2.tgz
```

Workaround for me now is to overwrite fp-pro in package.json, but that is not stable ofc.
```
  "dependencies": {
    "@fingerprintjs/fingerprintjs-pro-spa": "1.3.1",
    "@fingerprintjs/fingerprintjs-pro": "3.11.0"
  },
  "resolutions": {
    "@fingerprintjs/fingerprintjs-pro": "3.11.0"
  }
```
which gives me a yarn.lock file of
```
"@fingerprintjs/fingerprintjs-pro-spa@1.3.1":
  version "1.3.1"
  resolved "https://registry.yarnpkg.com/@fingerprintjs/fingerprintjs-pro-spa/-/fingerprintjs-pro-spa-1.3.1.tgz#4159fd57b03378703a08ff3bcc21b196939607af"
  integrity sha512-1k+85IYh1bBDJY/syHzBmTzKKaPGQkpq8GO9+vf7MHdqlmIiLAcutaHMUXey9DxylKm93qCHRY/1fCOOQ/LiTA==
  dependencies:
    "@fingerprintjs/fingerprintjs-pro" "^3.9.2"
    tslib "^2.6.2"

"@fingerprintjs/fingerprintjs-pro@3.11.0", "@fingerprintjs/fingerprintjs-pro@^3.9.2":
  version "3.11.0"
  resolved "https://registry.yarnpkg.com/@fingerprintjs/fingerprintjs-pro/-/fingerprintjs-pro-3.11.0.tgz#1ab6687010c36e2e01f5b5f60c3b7f6e75e2ef63"
  integrity sha512-93GjPzKPXoTj03Hvu9DA+P+py6ixQOXkOMLAzvIA7aY2uKoJQFgyCiL1ZFJvTsdml0gxy42rfkxf2z+AeRRo0w==
  dependencies:
    tslib "^2.4.1"
```

**Comment by ilfa** [comment]
Hey @Harm-Nullix!
Thank you for the issue, we will introduce new version of the `fingerprintjs-pro-spa` soon with updated dependency.

But it shouldn't block you, we declared dependency version as `"^3.9.2"` it means that `v3.11.0` is still a valid dependency.
To update it you can just call `npm update @fingerprintjs/fingerprintjs-pro-spa` or a similar command in your package manager, it will change lock file and update dependency.

**Comment by ilfa** [comment]
Released in [v1.3.2](https://github.com/fingerprintjs/fingerprintjs-pro-spa/releases/tag/v1.3.2)
